pom.xml

```diff
@@ -76,14 +72,13 @@
         <changelist>-SNAPSHOT</changelist>
         <jenkins.version>2.332.1</jenkins.version>
         <gitHubRepo>jenkinsci/${project.artifactId}</gitHubRepo>
-        <findbugs.failOnError>false</findbugs.failOnError>
```

Comment:
SpotBugs seems to be passing, so let us run it in local builds. CI already passes `failonError=false`.